README.md

```diff
@@ -68,6 +68,19 @@ To customise of appearance of StoriiiesViewer you have a few options:
 2. To 'theme' StoriiiesViewer, you may find the custom properties provided by the default stylesheet to be sufficient
 3. Start with default stylesheet and expand or override these styles as you see fit
 
+### Disabling Mouse Navigation
+
+If you want to disable interactions such as panning and zooming via scroll, you can pass the `disablePanAndZoom` option when initializing the viewer:
+
+````js
+document.addEventListener('DOMContentLoaded', () => {
+  const myViewer = new StoriiiesViewer({
+    container: "#storiiies-viewer",
+    manifestUrl: "https://path-to-your-storiiies-manifest",
+    disablePanAndZoom: true, // Disables mouse navigation
+  });
+});
+
```

Comment:
Small point: Think this should be a triple back-tick and closed off with triple back-ticks